client/view/new-deck/NewDeck.html

```diff
@@ -31,11 +31,14 @@ <h1>Add a new deck</h1>
 			Click "Publish".
 		</p>
 		<img srcset="images/google-spreadsheet-publish-example.png 2x" alt="A screenshot of the Google Sheets &quot;Publish to the Web&quot; dialog.">
+		<p>
+			Don't try to use the link that they give you here, it won't work.  It is very important that you share your spreadsheet first.
+		</p>
 		<p>
 			Close the "Publish to the Web" dialog.  Click the "Share" button at the top right.
 		</p>
 		<p>
-			Click "Get shareable link".  Then click the "Copy link" button.
+			Click "Get shareable link".
```

Comment:
Do you think it's clear enough without the "click the 'copy link'" message?

Reply:
When the user clicks "Get shareable link" it auto copies it to the clipboard.